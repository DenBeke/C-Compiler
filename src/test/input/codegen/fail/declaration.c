// Some undeclared test

int main() {
    a();
}