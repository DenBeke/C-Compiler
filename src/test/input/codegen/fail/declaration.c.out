[31m[ERROR] line 4: Use of undeclared function 'a'[0m
